lib/tableau/extensions/page_extension.ex

```diff
@@ -90,8 +90,12 @@ defmodule Tableau.PageExtension do
 
     pages =
       config.dir
-      |> Path.join("**/*.{#{exts}}")
-      |> Common.paths()
+      |> List.wrap()
```

Comment:
When I commented on adding this same functionality to the page extension, I meant in another PR, but it's also fine to add it here, just make sure you add a test case for the page extension as well.